packages/calypso-build/package.json

```diff
@@ -45,7 +45,7 @@
 		"@babel/preset-typescript": "7.3.3",
 		"@wordpress/babel-plugin-import-jsx-pragma": "2.3.0",
 		"@wordpress/browserslist-config": "2.3.0",
-		"@wordpress/dependency-extraction-webpack-plugin": "^1.0.1",
+		"@wordpress/dependency-extraction-webpack-plugin": "1.1.0",
```

Comment:
If you get blocked by eslint issue with removing deps, this change alone could be in separate PR.

Reply:
Do we need to pin this exact version?

Reply:
Isn't it the latest? (unless I misunderstand your question?)

Reply:
The question was more about removing the `^` - exact version vs. version range.

Not a big deal, it's a dependency that I can't imagine ever being duplicated into a bundle that's shipped to a browser. But the version range allows for deduplication. From memory, I don't think there have been significant changes since `1.0.1`